projects/js-packages/idc/components/card-fresh/index.jsx

```diff
@@ -107,23 +77,15 @@ CardFresh.propTypes = {
 	wpcomHomeUrl: PropTypes.string.isRequired,
 	/** The current site URL. */
 	currentUrl: PropTypes.string.isRequired,
-	/** The redirect URI to redirect users back to after connecting. */
-	redirectUri: PropTypes.string.isRequired,
-	/** Whether there's already an action in progress. */
-	isActionInProgress: PropTypes.bool,
-	/** Function to set the "action in progress" flag. */
-	setIsActionInProgress: PropTypes.func.isRequired,
+	/** Whether starting fresh is in progress. */
+	isStartingFresh: PropTypes.bool.isRequired,
```

Comment:
Should this prop be required? It has a default value. (I noticed a few other props in this PR that are both required and have a default value, but I didn't comment on each one in case this is intentional.)

Reply:
That's more of a personal preference. It can still be omitted with no warnings, because the default value is provided.
My thinking is that if the default value somehow gets removed, a warning will let developers know that the prop is missing. But that's an unlikely scenario.

Reply:
Thanks for the explanation! That makes sense.